ext/financialacls/financialacls.php

```diff
@@ -253,6 +254,38 @@ function financialacls_civicrm_membershipTypeValues($form, &$membershipTypeValue
   }
 }
 
+/**
+ * Add permissions.
+ *
+ * @see https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_permission/
+ *
+ * @param array $permissions
+ */
+function financialacls_civicrm_permission(&$permissions) {
+  if (!financialacls_is_acl_limiting_enabled()) {
+    return;
+  }
+  $actions = [
+    'add' => ts('add'),
```

Comment:
@eileenmcnaughton just a small thing this probably should be E::ts rather than ts given we are in an extension

Reply:
Yeah I guess so  - it works like that for core ext?

Reply:
Yes it does.